src/warnet/server.py

```diff
@@ -291,6 +293,28 @@ def down(self, network: str = "warnet") -> str:
             return f"Exception {e}"
 
 
+    def info(self, network: str = "warnet") -> str:
+        """
+        Get info about a warnet network named <network>
+        """
+        wn = Warnet.from_network(network)
+        return f"{wn}"
+
+
+    def status(self, network: str = "warnet") -> List[dict]:
+        """
+        Get running status of a warnet network named <network>
+        """
+        wn = Warnet.from_network(network)
+        stats = []
+        for tank in wn.tanks:
+            status = tank.container.status if tank.container is not None else None
```

Comment:
I kind of feel inclined here to use docker api to compare to our list of tanks, but it touches a lot of the other code and isn't worth the complexity (and yo've already got all this working)

Reply:
`container` here is retrieved from the API


https://github.com/bitcoin-dev-project/warnet/blob/080156dbd74fb85c2ca181227edf9a12bf5406a1/src/warnet/tank.py#L137-L144